openedx/core/djangoapps/agreements/tests/test_api.py

```diff
@@ -0,0 +1,100 @@
+"""
+Tests for the Agreements API
+"""
+import logging
+
+from testfixtures import LogCapture
+
+from common.djangoapps.student.tests.factories import UserFactory
+from openedx.core.djangoapps.agreements.api import (
+    create_integrity_signature,
+    get_integrity_signature,
+    get_integrity_signatures_for_course,
+)
+from openedx.core.djangolib.testing.utils import skip_unless_lms
+from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+LOGGER_NAME = "openedx.core.djangoapps.agreements.api"
+
+
+@skip_unless_lms
+class TestAgreementsApi(SharedModuleStoreTestCase):
```

Comment:
#NIT I think the name of this test suite can be updated to `TestIntegritySignatureAPI`. Agreements is the name of the app can that can be too over compassing for this test suite.